tests/update_workflow_test.go

```diff
@@ -1493,7 +1677,7 @@ func (s *integrationSuite) TestUpdateWorkflow_FailWorkflowTask() {
 	// Try to accept update in workflow: get malformed response.
 	_, err = poller.PollAndProcessWorkflowTask(false, false)
 	s.Error(err)
-	s.Equal(err.Error(), "BadUpdateWorkflowExecutionMessage: accepted_request is not set on update.Acceptance message body.")
+	s.Equal("BadUpdateWorkflowExecutionMessage: update some-random-wrong-id not found", err.Error())
```

Comment:
I use `some-random-wrong-id` with meaning that there is no dependency in test on this value. Here you introduce this dependency. Can you change `some-random-wrong-id` to `wrong-update-id` which will indicate that it is specific **wrong update Id**.